docs/roman/datamodels/models.rst

```diff
@@ -8,58 +8,84 @@ the underlying file format.  The same data model may be used for data
 created from scratch in memory, or loaded from ASDF files or some future file
 format.
 
+The detailed datamodel structure and specifics are contained in the
+documentation included with the roman_datamodels package found here (TBD).
 
-Hierarchy of models
--------------------
-
-There are different data model classes for different kinds of data.
-
-One model instance, many arrays
--------------------------------
-
-Each model instance generally has many arrays that are associated with
-it.  For example, the `ImageModel` class has the following arrays
-associated with it:
+Each model instance is created to contain a variety of attributes and data that
+are for needed for analysis or to propogate information about the file and the
+contents of the file. For example, the `ImageModel` class has the following
```

Comment:
just as a side note - when roman_datamodels documentaiton is up we should link to `ImageModel`.